contracts/StakeManager.sol

```diff
@@ -324,10 +348,10 @@ contract StakeManager is Ownable {
     /**
      * @dev Only callable from old manager.
      * @notice Increase total MP from old manager
-     * @param _increasedMP amount MP increased on account after migration initialized
+     * @param _amount amount MP increased on account after migration initialized
      */
-    function increaseMPFromMigration(uint256 _increasedMP) external onlyOldManager {
-        totalSupplyMP += _increasedMP;
+    function increaseTotalMP(uint256 _amount) external onlyPreviousManager {
```

Comment:
The entire "increasedMP" notion in this contract felt a bit off to me. 
In this case here, really what we're doing is adjusting the total MP of the next manager.